nixos/doc/manual/development/running-nixos-tests-interactively.xml

```diff
@@ -6,8 +6,7 @@
  <title>Running Tests interactively</title>
 
  <para>
-  The test itself can be run interactively. This is particularly useful when
-  developing or debugging a test:
+  The test itself can be run interactively. This is particularly useful when developing or debugging a test:
```

Comment:
Can you reflow/ indent these to match the context around them?

Reply:
The indent has been reformated that way by the docbook auto-formatter.

See the original post of this PR:

> Note: some diffs in the running-nixos-tests-interactively.xml file are coming from the docbook autoformater after running make. I decided to bit the bullet and these diffs, preventing any future reformatting. I can undo this if necessary.

Do we want to disable this format pass to keep the nice wrapping?

Reply:
Ok, let's wrap it.